Client/core/CSettings.cpp

```diff
@@ -381,6 +381,11 @@ void CSettings::CreateGUI()
     m_pCheckBoxAllowExternalSounds->GetPosition(vecTemp, false);
     m_pCheckBoxAllowExternalSounds->AutoSize(NULL, 20.0f);
 
+    m_pCheckBoxAlwaysShowTransferBox = reinterpret_cast<CGUICheckBox*>(pManager->CreateCheckBox(pTabMultiplayer, _("Always show download window"), false));
```

Comment:
Not sure if the description is clear enough for most players

Reply:
I assumed that even less players will understand what a "transfer box" is. I am open for suggestions for this setting.